README.md

```diff
@@ -34,6 +34,23 @@ The most important prefixes you should have in mind are:
 * `feat!:`,  or `fix!:`, `refactor!:`, etc., which represent a breaking change
   (indicated by the `!`) and will result in a SemVer major.
 
+## How do I change the version number?
+
+This will only work if Release Please has been installed as a GitHub Action.
```

Comment:
I think this should work for the app too, if it failed in the past it might have just been that we hadn't deployed the latest version of the tool to Cloud Functions.